plugin/helpers.go

```diff
@@ -29,8 +33,23 @@ type Helpers interface {
 	//
 	// Minimum server version: 5.6
 	KVSetWithExpiryJSON(key string, value interface{}, expireInSeconds int64) error
+
+	// RegisterCommand registers the given command and callback for the ExecuteCommand helper.
+	// Returns error if the callback is not provided or the API.RegisterCommand fails
+	RegisterCommand(command *model.Command, callback CommandCallback) error
+
+	// ExecuteCommand handles command execution, triggering previously registered callbacks.
+	// It is meant to replace the implementation of the hook of the same name:
+	//     func (p *Plugin) ExecuteCommand(c *Context, args *model.CommandArgs) {
+	//         return p.Helpers.ExecuteCommand(c, args)
+	//     }
+	// Returns (&model.CommandResponse{}, nil) if for some reason the callback doesn't exist
```

Comment:
I'm not sure it makes sense to describe all the return values here -- this is basically just duplicating the code, and it's very likely to get out of date. Propose removing all the `Returns ...` comments.

Reply:
I've seen in the other comments and I just follow a bit the idea. Is ok for me to remove them